docs/checkit.js

```diff
@@ -32,7 +32,13 @@ function CheckIt(mode, calendarDiv) {
     this.$endDate = $('#endDate');
     this.$buildCalendarForm = $('#collapseOne');
     this.$loadingWheel = $('#loadingWheel');
-    this.$calendarDiv = calendarDiv;
+    
+    if (calendarDiv.length === 0) {
+        console.error("Selector given for calendar div does not select anything.");
+    }
+    else {
+        this.$calendarDiv = calendarDiv;
```

Comment:
I would always do this. Either you do what the client asked you to do (assign this value) or you refuse to do anything at all. The worst thing to do is to magically change your behavior based on a best guess about what should happen. You've warned them that what is about to happens seems stupid. That's enough.